cmd/acr/purge.go

```diff
@@ -209,7 +210,7 @@ func ParseDuration(ago string) (time.Duration, error) {
 // returns a pointer to a slice that contains the tags that will be deleted, the last tag obtained through the AcrListTags method
 // and an error in case it occurred, the fourth return value contains a map that is used to determine how many tags a manifest has
 func GetTagsToDelete(ctx context.Context,
```

Comment:
>GetTagsToDelete [](start = 5, length = 15)

Do you need to export the functions in the module? If not, should you just rename them, eg, getTagsToDelete

Reply:
All right, I've changed them all in the purge.go file, in the other ones I think they were unexported.